lock-impl/src/test/java/com/palantir/lock/impl/ClientAwareLockTest.java

```diff
@@ -73,85 +75,46 @@ public void setUp() {
     /** Tests using an anonymous (non-reentrant) client. */
     @Test
     public void testAnonymousClient() throws InterruptedException {
-        Assert.assertNull(anonymousReadLock.tryLock());
-        Assert.assertNotNull(anonymousWriteLock.tryLock());
-        Assert.assertNotNull(anonymousWriteLock.tryLock(10, TimeUnit.MILLISECONDS));
+        assertThat(anonymousReadLock.tryLock()).isNull();
+        assertThat(anonymousWriteLock.tryLock()).isNotNull();
+        assertThat(anonymousWriteLock.tryLock(10, TimeUnit.MILLISECONDS)).isNotNull();
         anonymousReadLock.lock();
         anonymousReadLock.unlock();
         anonymousReadLock.unlock();
-        Assert.assertNull(anonymousWriteLock.tryLock(10, TimeUnit.MILLISECONDS));
-        Assert.assertNotNull(anonymousReadLock.tryLock(10, TimeUnit.MILLISECONDS));
-        Assert.assertNotNull(anonymousWriteLock.tryLock());
+        assertThat(anonymousWriteLock.tryLock(10, TimeUnit.MILLISECONDS)).isNull();
+        assertThat(anonymousReadLock.tryLock(10, TimeUnit.MILLISECONDS)).isNotNull();
+        assertThat(anonymousWriteLock.tryLock()).isNotNull();
         anonymousWriteLock.unlock();
     }
 
     /** Tests that things fail when they should. */
     @Test
     public void testIllegalActions() throws InterruptedException {
```

Comment:
In general for this test, probably useful to retain _why_ the behaviours are expected (e.g. you can do this via an `as(...)` statement on the assertion. For example, the first two could be something like `"should not be able to unlock when no locks are held"`